### PR TITLE
Updated load test file for /v1/etor/orders endpoint

### PR DESCRIPTION
**Pull request by ghost**
# Updated load test file for /v1/etor/orders endpoint

- added test for /v1/etor/orders endpoint

## Issue #382 

- [x] I have added tests to cover my changes

### CODE REVIEW

File: e2e/src/test/resources/lab_order.json
Comment (tjohnson7021) [inline e2e/src/test/resources/lab_order.json:1]: @halprin I used the lab_order.json file as the order request body data for the load test. I made this assumption based on some questions I asked prior.

**Comment by tjohnson7021** [comment]
I put this in draft mode in the event that I am missing something significant.

File: operations/locustfile.py [hunk 1]
```diff
@@ -96,3 +105,9 @@ def get_demographics_request_body():
     # read the sample request body for the demographics endpoint
     with open("e2e/src/test/resources/newborn_patient.json", "r") as f:
         return f.read()
+
+
+def get_orders_request_body():
```
Comment (basiliskus) [inline operations/locustfile.py:110]: Looks good, it's just missing the instantiation of `order_request_body = get_orders_request_body()` inside `test_start`

**Review by basiliskus — COMMENTED** [review]
Good job! It looks good to me. I'd just defer approval so someone else can validate the order data bundle

**Comment by tjohnson7021** [comment]
i'm not sure why this [git check](https://github.com/CDCgov/trusted-intermediary/actions/runs/5425745053/jobs/9866974945?pr=413) keeps failing...

**Review by halprin — APPROVED** [review]
Looking good!  Where did you get the `lab_order.json` from?  Also yes, strange that the GitHub action fails.

**Pull request merged**